pkg/test/auth/tokenmanager.go

```diff
@@ -69,21 +69,21 @@ type ExtraClaim func(token *jwt.Token)
 // WithEmailClaim sets the `email` claim in the token to generate
 func WithEmailClaim(email string) ExtraClaim {
 	return func(token *jwt.Token) {
-		token.Claims.(jwt.MapClaims)["email"] = email
+		token.Claims.(*MyClaims).Email = email
 	}
 }
 
 // WithIATClaim sets the `iat` claim in the token to generate
 func WithIATClaim(iat time.Time) ExtraClaim {
```

Comment:
We can probably just remove this now since you've provided an alternative!

Reply:
I think we still need this, without it there's no easy way in our tests to override the default claim values assigned to the token.